monitor/src/main/java/edu/asu/diging/monitor/web/admin/AddAppController.java

```diff
@@ -39,14 +50,38 @@ public String show(Model model) {
             recipient.setEmail(r.getEmail());
             return recipient;
         }).collect(Collectors.toList()));
+        appForm.setGroupIds(groupManager.getGroups().stream()
+                .collect(Collectors.toMap(g -> g.getId(), g -> g.getName())).entrySet());
         return "admin/apps/show";
     }
-	
-	@RequestMapping(value="/admin/apps/add", method=RequestMethod.POST)
-	public String add(@ModelAttribute AppForm appForm) {
-		IApp app = new App();
-		appHelper.copyAppInfo(app, appForm);
-		appManager.addApp(app);
-		return "redirect:/admin/apps/add";
-	}
+
+    @RequestMapping(value = "/admin/apps/add", method = RequestMethod.POST)
+    public String add(@ModelAttribute AppForm appForm, RedirectAttributes redirectAttrs) {
+        IApp app = new App();
+        if (appForm.getGroupType() == null
```

Comment:
from the ticket:
"It should be possible to not assign a group to an app though (in other words, apps do not have to in a group)."